addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -69,3 +75,84 @@ test("editing livechat note is synced between tabs", async () => {
+        user_ids: [userId],
+    });
+    const countryId = pyEnv["res.country"].create({ code: "be", name: "Belgium" });
+    const guestId = pyEnv["mail.guest"].create({
+        name: "Visitor #20",
+    });
+    const channelId = pyEnv["discuss.channel"].create({
+        anonymous_name: "Visitor #20",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+        ],
+        country_id: countryId,
+        channel_type: "livechat",
+        livechat_operator_id: serverState.partnerId,
+        livechat_status: "waiting",
+    });
+    await start();
+    await openDiscuss(channelId);
+    await waitFor(".o-livechat-ChannelInfoList button.active", { text: "Waiting for customer" });
+    await waitFor(
+        ".o-mail-DiscussSidebar-item span[title='Waiting for customer'] i.fa-circle.o-warning"
+    );
+    await click(".o-livechat-ChannelInfoList button", { text: "Looking for help" });
+    await waitFor(".o-livechat-ChannelInfoList button.active", { text: "Looking for help" });
+    await waitFor(".o-mail-DiscussSidebar-item span[title='Looking for help'] i.fa-circle.o-info");
+});
+
+test("editing livechat status is synced between tabs", async () => {
+    const pyEnv = await startServer();
+    const userId = pyEnv["res.users"].create({ name: "James" });
+    pyEnv["res.partner"].create({
+        name: "James",
+        user_ids: [userId],
+    });
+    const countryId = pyEnv["res.country"].create({ code: "be", name: "Belgium" });
+    const guestId = pyEnv["mail.guest"].create({
+        name: "Visitor #20",
+    });
+    const channelId = pyEnv["discuss.channel"].create({
+        anonymous_name: "Visitor #20",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+        ],
+        country_id: countryId,
+        channel_type: "livechat",
+        livechat_operator_id: serverState.partnerId,
+        livechat_status: "in_progress",
+    });
+    const tab1 = await start({ asTab: true });
+    const tab2 = await start({ asTab: true });
+    await openDiscuss(channelId, { target: tab1 });
+    await openDiscuss(channelId, { target: tab2 });
+    await waitFor(".o-livechat-ChannelInfoList button.active", {
+        text: "In progress",
+        target: tab1,
+    });
+    await waitFor(".o-livechat-ChannelInfoList button.active", {
+        text: "In progress",
+        target: tab2,
+    });
+    await click(".o-livechat-ChannelInfoList button", {
+        text: "Waiting for customer",
+        target: tab1,
+    });
+    await waitFor(".o-livechat-ChannelInfoList button.active", {
+        text: "Waiting for customer",
+        target: tab1,
+    });
+    await waitFor(".o-livechat-ChannelInfoList button.active", {
+        text: "Waiting for customer",
+        target: tab2,
+    }); // Status should be synced with bus
```

Comment:
`waitFor` doesn't have `text` as said before so silently ignored.

`target: tab` is also not supported (that's specific to our `contains()`), thus it's also silently ignored.
I think should use `root: tab.target` for the equivalent intend with `waitFor()`.